.github/jobs/GithubRelease.pkl

```diff
@@ -0,0 +1,37 @@
+import "@gha/actions/Artifact.pkl"
+import "@gha/Context.pkl"
+import "@gha/Workflow.pkl"
+
+fixed job: Workflow.Job = new {
+  `runs-on` = "ubuntu-24.04"
```

Comment:
Is there a reason not to use `ubuntu-latest` here? The specific version names are already "un-pinned" in that github can and does change them.

Reply:
Yeah, good point!